js/views/userCard.js

```diff
@@ -4,19 +4,38 @@ import loadTemplate from '../utils/loadTemplate';
 import app from '../app';
 import { followedByYou, followUnfollow } from '../utils/follow';
 import Profile from '../models/profile/Profile';
+import UserCard from '../models/UserCard';
 
 export default class extends BaseVw {
   constructor(options = {}) {
     super(options);
     this.options = options;
-    this.guid = options.guid || this.model.get('guid');
+
+    if (this.model instanceof Profile) {
+      this.guid = this.model.id;
+      this.fetched = true;
+    } else if (this.model) {
+      this.guid = this.model.get('guid');
+      this.fetched = false;
+    } else {
+      this.model = new UserCard({ guid: options.guid });
```

Comment:
When userCard views are created by the followers collection, the model on construction is userCard, which triggers the creation and fetching of the profile model. If only a guid option was passed in, this keeps that structure. 

We could probably refactor all of it to always use profile models instead, but that might be better as a separate task.